Add Province for Indonesia to database Magento 2.1

jurpro opened the pull request:
Hello.

How to request add province to Magento for officially use ? 

Here is province in Indonesia.

Country : Indonesia / IDN / ID

I. For add database in: directory_country_region
"512","ID","ACEH","Aceh"
"513","ID","SUMUT","Sumatera Utara"
"514","ID","SUMBAR","Sumatera Barat"
"515","ID","RIAU","Riau"
"516","ID","JAMBI","Jambi"
"517","ID","SUMSEL","Sumatera Selatan"
"518","ID","BENGKULU","Bengkulu"
"519","ID","LAMPUNG","Lampung"
"520","ID","KEP_BABEL","Kepulauan Bangka Belitung"
"521","ID","KEP_RIAU","Kepulauan Riau"
"522","ID","DKI_JAKARTA","DKI Jakarta"
"523","ID","JABAR","Jawa Barat"
"524","ID","JATENG","Jawa Tengah"
"525","ID","DIY","D.I Yogyakarta"
"526","ID","JATIM","Jawa Timur"
"527","ID","BANTEN","Banten"
"528","ID","BALI","Bali"
"529","ID","NTB","Nusa Tenggara Barat"
"530","ID","NTT","Nusa Tenggara Timur"
"531","ID","KALBAR","Kalimantan Barat"
"532","ID","KALTENG","Kalimantan Tengah"
"533","ID","KALSEL","Kalimantan Selatan"
"534","ID","KALTIM","Kalimantan Timur"
"535","ID","KALTARA","Kalimantan Utara"
"536","ID","SULUT","Sulawesi Utara"
"537","ID","SULTENG","Sulawesi Tengah"
"538","ID","SULSEL","Sulawesi Selatan"
"539","ID","SULTRA","Sulawesi Tenggara"
"540","ID","GORONTALO","Gorontalo"
"541","ID","SULBAR","Sulawesi Barat"
"542","ID","MALUKU","Maluku"
"543","ID","MALUT","Maluku Utara"
"544","ID","PAPUA","Papua"
"545","ID","PAPUA_BARAT","Papua Barat"

II. For add database in: directory_country_region_name
"en_US","512","Aceh"
"en_US","513","Sumatera Utara"
"en_US","514","Sumatera Barat"
"en_US","515","Riau"
"en_US","516","Jambi"
"en_US","517","Sumatera Selatan"
"en_US","518","Bengkulu"
"en_US","519","Lampung"
"en_US","520","Kepulauan Bangka Belitung"
"en_US","521","Kepulauan Riau"
"en_US","522","DKI Jakarta"
"en_US","523","Jawa Barat"
"en_US","524","Jawa Tengah"
"en_US","525","D.I Yogyakarta"
"en_US","526","Jawa Timur"
"en_US","527","Banten"
"en_US","528","Bali"
"en_US","529","Nusa Tenggara Barat"
"en_US","530","Nusa Tenggara Timur"
"en_US","531","Kalimantan Barat"
"en_US","532","Kalimantan Tengah"
"en_US","533","Kalimantan Selatan"
"en_US","534","Kalimantan Timur"
"en_US","535","Kalimantan Utara"
"en_US","536","Sulawesi Utara"
"en_US","537","Sulawesi Tengah"
"en_US","538","Sulawesi Selatan"
"en_US","539","Sulawesi Tenggara"
"en_US","540","Gorontalo"
"en_US","541","Sulawesi Barat"
"en_US","542","Maluku"
"en_US","543","Maluku Utara"
"en_US","544","Papua"
"en_US","545","Papua Barat"

omiroshnichenko committed:
MAGETWO-59416: [Backport] - [Github] Admin can't reset password for more than one customer #5260 - for 2.1

Commit:
MAGETWO-58437: [Backport] [github #6195, 4101] Gallery doesn't show all images added to configurable options - for 2.1.x

Commit:
MAGETWO-59142: [Backport] UI Grid won't load initial data until its state gets modified

Commit:
MAGETWO-57343: App Config Reader

viktym committed:
MAGETWO-59837: Stabilization

- Fix Magento\Vault\Test\TestCase\UseVaultOnCheckoutTest
- Fix Magento\Vault\Test\TestCase\DeleteSavedCreditCardTest
- Fix Magento\Vault\Test\TestCase\CreateVaultOrderBackendTest

viktym committed:
MAGETWO-59124: [Backport] - [GITHUB] Credit Card capture not associated with the Authorization since upgrade to 2.1.1 #6716

YevSent committed:
MAGETWO-59837: Stabilization for ported Vault and Braintree changes

 - Added missed configuration and constraints

YevSent committed:
Merge remote-tracking branch 'origin/MAGETWO-59578' into MAGETWO-59578

Commit:
MAGETWO-57343: App Config Reader

iivashchenko committed:
MAGETWO-60036: Port fix for OnePageCheckoutWithDiscountTest

- OnePageCheckoutWithDiscountTest included

Commit:
MAGETWO-57343: App Config Reader

valdislav committed:
Merge branch '2.1-develop' into MAGETWO-60053

valdislav committed:
Merge branch '2.1-develop' of github.com:magento/magento2ce into MAGETWO-58977

valdislav committed:
Merge branch 'MAGETWO-58500' of github.com:magento-south/magento2ce into PR-2.1-24102016

valdislav committed:
Merge branch 'MAGETWO-60053' into PR-2.1-24102016

Commit:
Merge pull request #543 from magento-okapis/MAGETWO-58335-Uploaded-Image-2.1.3

Fixed issue:
 - MAGETWO-58335 [Backport] - Can't view uploaded image

xmav committed:
MAGETWO-60103: [Backport] Configurable variation is displayed on category/product page when is out of stock - 2.1

Commit:
MAGETWO-59953: [Backport] Configurable product option price is displayed incorrectly per website for 2.1.3

- fix unit tests

valdislav committed:
Merge branch '2.1-develop' of github.com:magento/magento2ce into PR-2.1-24102016

Commit:
Merge remote-tracking branch 'origin/2.1-develop' into PR-2.1-24102016

Commit:
Merge branch 'PR-2.1-24102016' of github.com:magento-firedrakes/magento2ce into PR-2.1-24102016

YevSent committed:
MAGETWO-59837: Stabilization for ported Vault and Braintree changes

 - Added waiting for Braintree PayPal button

Commit:
Merge pull request #544 from magento-firedrakes/PR-2.1-24102016

[Firedrakes] Bugfixes 2.1

Commit:
Merge remote-tracking branch 'mainline/2.1-develop' into MAGETWO-59953

YevSent committed:
Merge branch '2.1-develop' of https://github.com/magento/magento2ce into MAGETWO-59578

xmav committed:
MAGETWO-60103: [Backport] Configurable variation is displayed on category/product page when is out of stock - 2.1

xmav committed:
MAGETWO-60103: [Backport] Configurable variation is displayed on category/product page when is out of stock - 2.1

Commit:
MAGETWO-58895: [Backport] Customer is redirected to "Compare Products" Frontend page if tries to remove a Product from comparing

 - back ported fix that doesn't redirect users when they remove product from compare list

xmav committed:
MAGETWO-60103: [Backport] Configurable variation is displayed on category/product page when is out of stock - 2.1

xmav committed:
MAGETWO-60103: [Backport] Configurable variation is displayed on category/product page when is out of stock - 2.1

xmav committed:
MAGETWO-60103: [Backport] Configurable variation is displayed on category/product page when is out of stock - 2.1

xmav committed:
MAGETWO-60103: [Backport] Configurable variation is displayed on category/product page when is out of stock - 2.1

Commit:
Merge remote-tracking branch 'mainline/2.1-develop' into MAGETWO-57343

xmav committed:
MAGETWO-60103: [Backport] Configurable variation is displayed on category/product page when is out of stock - 2.1

Commit:
MAGETWO-60060: Magento 2.1.3 Publication

Commit:
MAGETWO-57343: App Config Reader

slavvka committed:
Merge pull request #549 from magento-frontend/MAGETWO-57343

[Frontend] MAGETWO-57343 [Backport] - Deployment process can be executed on separate machine - for 2.1

Commit:
Merge remote-tracking branch 'mainline/2.1-develop' into MAGETWO-59953

YevSent committed:
Merge branch '2.1-develop' of https://github.com/magento/magento2ce into MAGETWO-59578

slavvka committed:
Merge pull request #550 from magento-dragons/MAGETWO-59953

[Dragons] Bug fixing - 2.1:
- MAGETWO-59953 [Backport] Configurable product option price is displayed incorrectly per website for 2.1.3
- MAGETWO-60103 [[Backport] Configurable variation is displayed on category/product page when is out of stock - 2.1

viktym committed:
MAGETWO-59837: Stabilization for ported Vault and Braintree changes

- Return removed constructor arguments and PAYPAL_CODE constant

viktym committed:
MAGETWO-59837: Stabilization for ported Vault and Braintree changes

- Update constructor annotation

Commit:
MAGETWO-58312: [Backport][GitHub] Products became “out of stock” after update from 2.0.7 to 2.1.0 #5222 - for 2.1

viktym committed:
MAGETWO-59837: Stabilization for ported Vault and Braintree changes

- Update Conflict Resolution test

Commit:
MAGETWO-56804: [Backport] Product catalog Import/export - Date & Timezone issue - for 2.1

 - backport changes to fix timezone issues with import/export

Commit:
MAGETWO-56804: [Backport] Product catalog Import/export - Date & Timezone issue - for 2.1

 - backport changes to fix timezone issues with import/export

Commit:
Merge branch 'MAGETWO-58312-Products-Out-Of-Stock' into MAGETWO-60060-Release-Publication

Commit:
Merge branch 'MAGETWO-56804_importexport_timezone_2.1.x' into MAGETWO-60060-Release-Publication

Commit:
MAGETWO-59581: Merge PayPal Ask MAGETWO-53942 to 2.1.x

- Braintree help links added

Commit:
Merge pull request #553 from magento-okapis

Fixed issue:
 - MAGETWO-56804 [Backport] Product catalog Import/export - Date & Timezone issue - for 2.1
 - MAGETWO-58312 [Backport][GitHub] Products became “out of stock” after update from 2.0.7 to 2.1.0 #5222 - for 2.1

viktym committed:
MAGETWO-59837: Stabilization for ported Vault and Braintree changes

- Fix Conflict Resolution test

viktym committed:
Merge remote-tracking branch 'origin/MAGETWO-59578' into MAGETWO-59578

mslabko committed:
Merge remote-tracking branch 'mainline/2.1-develop' into MAGETWO-60041

Commit:
MAGETWO-60348: Remove api annotation

Commit:
MAGETWO-60364: Prepare code base for 2.1.4-dev release

Commit:
Merge pull request #555 from magento-dragons/MAGETWO-60348

[Dragons] MAGETWO-60348: Remove api annotation - 2.1

YevSent committed:
MAGETWO-59837: Stabilization for ported Vault and Braintree changes

 - Added payment code for event triggering

Commit:
Merge pull request #556 from magento-okapis

MAGETWO-60364: Prepare code base for 2.1.4-dev release

shiftedreality committed:
MAGETWO-59809: Scrub sensitive data during app:dump

Commit:
MAGETWO-60355: [Backport] - Production mode cannot be enabled - for 2.1

YevSent committed:
Merge branch 'MAGETWO-59578' into MPI-PR-S82

oshmyheliuk committed:
MAGETWO-60381: Disable fields functionality doesn't work for theme configuration

shiftedreality committed:
MAGETWO-59809: Scrub sensitive data during app:dump

shiftedreality committed:
Merge branch 'MAGETWO-59376' of https://github.com/magento-falcons/magento2ce into MAGETWO-59376

oshmyheliuk committed:
MAGETWO-60381: Disable fields functionality doesn't work for theme configuration

oshmyheliuk committed:
MAGETWO-60381: Disable fields functionality doesn't work for theme configuration

Commit:
MAGETWO-60355: [Backport] - Production mode cannot be enabled - for 2.1

Commit:
Merge branch 'MAGETWO-59376' of https://github.com/magento-falcons/magento2ce into MAGETWO-59376

oshmyheliuk committed:
Merge remote-tracking branch 'falcon/MAGETWO-59376' into MAGETWO-60381

shiftedreality committed:
MAGETWO-59809: Scrub sensitive data during app:dump

shiftedreality committed:
Merge branch 'MAGETWO-59376' of https://github.com/magento-falcons/magento2ce into MAGETWO-59376

Commit:
MAGETWO-60366: Cannot Enable Production Mode

Commit:
MAGETWO-60366: Cannot Enable Production Mode

Commit:
Merge pull request #562 from magento-mpi/MPI-PR-S82

Fixed issue:
 - MAGETWO-59124 [Backport] - [GITHUB] Credit Card capture not associated with the Authorization since upgrade to 2.1.1 #6716 - for 2.1
 - MAGETWO-59578 Merge PayPal Ask MAGETWO-40517 and MAGETWO-52475 to 2.1.x
 - MAGETWO-59581 Merge PayPal Ask MAGETWO-53942 to 2.1.x
 - MAGETWO-59894 [Backport] Email Associated with PayPal Merchant Account Adaptation for On-Boarding

Commit:
MAGETWO-60366: Cannot Enable Production Mode

Commit:
MAGETWO-60366: Cannot Enable Production Mode

Commit:
Merge pull request #563 from magento-okapis

Fixed issues:
 - MAGETWO-60366 Cannot Enable Production Mode
 - MAGETWO-60041 [Backport] Resize images mechanism is broken on frontend 2.1.3

shiftedreality committed:
Merge branch '2.1' of https://github.com/magento/magento2ce into MAGETWO-59376

# Conflicts:
#	app/code/Magento/Braintree/etc/di.xml
#	app/code/Magento/Paypal/etc/di.xml

shiftedreality committed:
MAGETWO-60145: Mainline Build Failure for Sample Data Tests on Version 2.1-develop

Commit:
MAGETWO-60460: Remove minor changes in Payment and Vault API

Commit:
MAGETWO-60460: Remove minor changes in Payment and Vault API

 - MAGETWO-60471: Remove ApiAnnotationTest for API namespace

Commit:
Merge pull request #567 from magento-mpi/MAGETWO-60460

MAGETWO-60460: Remove minor changes in Payment and Vault API

Commit:
Merge pull request #566 from magento-falcons/MAGETWO-59376

Fixed issue: 
- Split Deployment - scrub sensitive data - override config values by environment

Commit:
MAGETWO-60477: Invalid SalesInventory module version - 2.1

Commit:
Merge pull request #569 from magento-okapis/MAGETWO-60477-Sales-Inventory

MAGETWO-60477: Invalid SalesInventory module version - 2.1

ameysar committed:
MAGETWO-56695: Paypal checkout not possible with products that have custom options #5938 - for 2.1

RomaKis committed:
MAGETWO-58893: [Backport] [GitHub] IndexerHandlerFactory: Indexer Object cast to String - 2.1

ameysar committed:
MAGETWO-58376: PayPal Payflow Pro always using USD (even if this currency is absent on your store) - for 2.1

RomaKis committed:
MAGETWO-59036: [Backport] [GITHUB] Broken File Type Custom Option #5434 - 2.1

eug123 committed:
Merge remote-tracking branch 'origin/MAGETWO-58437' into pr-2.1-develop

eug123 committed:
Merge remote-tracking branch 'origin/MAGETWO-59142' into pr-2.1-develop

zakdma committed:
MAGETWO-58832: [Backport] Order comments history shows time of comment twice - for 2.1.x

Commit:
Merge remote-tracking branch 'origin/MAGETWO-59416' into bugfixes_2.1.4

Commit:
Merge remote-tracking branch 'origin/MAGETWO-57832' into bugfixes_2.1.4

Commit:
Merge remote-tracking branch 'origin/MAGETWO-58559' into bugfixes_2.1.4

MomotenkoNatalia committed:
Merge pull request #571 from magento-falcons/MAGETWO-60145

Fixed issue: 
- MAGETWO-60145: Mainline Build Failure for Sample Data Tests

ameysar committed:
MAGETWO-56695: Paypal checkout not possible with products that have custom options #5938 - for 2.1

Commit:
MAGETWO-55612: “No Payment method available” when customer tries to ship his items to billing restricted country for 2.1.x

eug123 committed:
Merge remote-tracking branch 'mainline/2.1-develop' into pr-2.1-develop

Commit:
Merge remote-tracking branch 'mainline/2.1-develop' into bugfixes_2.1.4

ameysar committed:
Merge remote-tracking branch 'origin/MAGETWO-55612' into 2.1-develop-pr1

ameysar committed:
Merge remote-tracking branch 'origin/MAGETWO-58376' into 2.1-develop-pr1

ameysar committed:
Merge remote-tracking branch 'origin/MAGETWO-58893' into 2.1-develop-pr1

Commit:
Merge branch 'MAGETWO-59036' into 2.1-develop-pr1

OlgaVasyltsun committed:
MAGETWO-60248: Billing Address and Default Shipping Address checkboxes on Customer page are saved incorrectly

OlgaVasyltsun committed:
MAGETWO-60248: Billing Address and Default Shipping Address checkboxes on Customer page are saved incorrectly

Commit:
MAGETWO-55612: “No Payment method available” when customer tries to ship his items to billing restricted country for 2.1.x

Commit:
Merge branch 'MAGETWO-55612' into 2.1-develop-pr1

Commit:
Merge pull request #579 from magento-firedrakes/bugfixes_2.1.4

[Firedrakes] Bugfixes 2.1.4

ameysar committed:
Merge remote-tracking branch 'origin/MAGETWO-60248' into 2.1-develop-pr1

Commit:
Merge branch 'MAGETWO-58832' into 2.1-develop-pr1

eug123 committed:
MAGETWO-58437: [Backport] [github #6195, 4101] Gallery doesn't show all images added to configurable options - for 2.1.x

Merge remote-tracking branch 'mainline/2.1-develop' into pr-2.1-develop

# Conflicts:
#	app/code/Magento/Swatches/view/frontend/web/js/swatch-renderer.js

Commit:
Merge pull request #584 from magento-frontend/pr-2.1-develop

[Frontend] Bugs - for 2.1.x

zakdma committed:
Merge branch '2.1-develop' into 2.1-develop-pr1

Commit:
Merge pull request #593 from magento-tsg/2.1-develop-pr1

[TSG] Backporting for 2.1

Commit:
MAGETWO-60479: Importing existing products with 'Replace' behavior gives no errors and deletes them

Commit:
MAGETWO-60479: Importing existing products with 'Replace' behavior gives no errors and deletes them

 - add tests

Commit:
MAGETWO-60898: Table vault_payment_token doesn't exist

Commit:
MAGETWO-61039: Error when saving an existing product with an image

Commit:
Merge pull request #607 from magento-mpi/MAGETWO-60898

Fixed issue:
 - MAGETWO-60898: Table vault_payment_token doesn't exist

Commit:
MAGETWO-60479: Importing existing products with 'Replace' behavior gives no errors and deletes them

 - Merge remote-tracking branch 'mainline/2.1' into MAGETWO-60479-Import-Products

Commit:
Merge pull request #612 from magento-okapis/MAGETWO-60479-Import-Products

[Okapis] MAGETWO-60479: Importing existing products with 'Replace' behavior

Commit:
MAGETWO-60293: [Backport] - [GitHub][IE]Checkout cart totals.js throws an error when estimating shipping - for 2.1 #5358 #7051

Commit:
MAGETWO-61039: Error when saving an existing product with an image

fixed unit-test

xmav committed:
MAGETWO-60605: Exception when adding configurable product by sku from customer account if associated simple product is out of stock

igrybkov committed:
MAGETWO-61055: [Backport] - Catalog broken when all child products of configurable are disabled - for 2.1.3

Commit:
MAGETWO-60832: UI Upgrade Fails from CE 2.1.1 to EE 2.1.3

slavvka committed:
Merge pull request #622 from magento-frontend/MAGETWO-60832

[Frontend] MAGETWO-60832: UI Upgrade Fails from CE 2.1.1 to EE 2.1.3

igrybkov committed:
MAGETWO-60605: Exception when adding configurable product by sku from customer account if associated simple product is out of stock

igrybkov committed:
MAGETWO-61055: [Backport] - Catalog broken when all child products of configurable are disabled - for 2.1.3

Commit:
MAGETWO-54704: [Backport] [GITHUB] Product prices not scoped to Website - scoping seems to be at store view level #5133 #7251 - for 2.1

Commit:
MAGETWO-56998: [Backport] Simple child product without a special price still shown as "was (original price)" #4442 #5097 #6645 - for 2.1

igrybkov committed:
MAGETWO-60605: Exception when adding configurable product by sku from customer account if associated simple product is out of stock

Commit:
Merge pull request #632 from magento-dragons/dragons-pr-2.1

[DRAGONS] Bugfixes for 2.1.3

slavvka committed:
Merge remote-tracking branch 'origin/MAGETWO-56998' into BUGS-2.1

Commit:
MAGETWO-60890: Fatal error logging in as admin user with restricted role

Commit:
Merge pull request #634 from magento-dragons/MAGETWO-54704

MAGETWO-54704: [Backport] [GITHUB] Product prices not scoped to Website

Commit:
MAGETWO-60890: Fatal error logging in as admin user with restricted role

Commit:
MAGETWO-60890: Fatal error logging in as admin user with restricted role

Commit:
Merge branch '2.1' of github.com:magento/magento2ce into MAGETWO-61039

slavvka committed:
Merge pull request #635 from magento-south/BUGS-2.1

[South] Bugs 2.1:
- MAGETWO-56998 [Backport] Simple child product without a special price still shown as "was (original price)" #4442 #5097 #6645 - for 2.1

Commit:
Merge branch '2.1' of github.com:magento/magento2ce into MAGETWO-61039

Commit:
Merge pull request #638 from magento-east/MAGETWO-61039

MAGETWO-61039: Error when saving an existing product with an image

Commit:
Merge pull request #625 from magento-troll/pullrequest-2.1

[Troll] Bugfix Backport

Commit:
MAGETWO-60890: Fatal error logging in as admin user with restricted role

Commit:
MAGETWO-60890: Fatal error logging in as admin user with restricted role

Commit:
MAGETWO-60890: Fatal error logging in as admin user with restricted role

Commit:
Merge remote-tracking branch 'mainline/2.1' into MAGETWO-60890

Commit:
MAGETWO-60890: Fatal error logging in as admin user with restricted role

Commit:
MAGETWO-60890: Fatal error logging in as admin user with restricted role

slavvka committed:
Merge pull request #643 from magento-south/MAGETWO-60890

[South] Magetwo 60890: Fatal error logging in as admin user with restricted role

Commit:
Merge branch '2.1-develop' into MAGETWO-58895-compare_products_2.1.x

Commit:
MAGETWO-61850: UI Upgrade Test failure when no other components to upgrade

 - changed CSS selector for other components page
 - added new events preset to improve logging for UI test

Commit:
MAGETWO-61850: UI Upgrade Test failure when no other components to upgrade

  - code review changes

Commit:
MAGETWO-61850: UI Upgrade Test failure when no other components to upgrade

  - renamed upgrade events_preset to debug

Commit:
MAGETWO-61850: UI Upgrade Test failure when no other components to upgrade

  - added SourceCode logging for debug events preset

Commit:
Merge pull request #66 from magento-fearless-kiwis/MAGETWO-61850-2.1-upgrade-test

MAGETWO-61850: UI Upgrade Test

Commit:
Merge pull request #650 from magento-fearless-kiwis/MAGETWO-61850-2.1-upgrade-test

MAGETWO-61850: UI Upgrade Test

magento-cicd2 commented:
[![CLA assistant check](https://cla.dev.magento.com/pull/badge/not_signed)](https://cla.dev.magento.com/magento/magento2?pullRequest=7695) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/magento2?pullRequest=7695) before we can accept your contribution.<br/>**2** out of **12** committers have signed the CLA.<br/><br/>:white_check_mark: okorshenko<br/>:white_check_mark: voleye<br/>:x: sereban<br/>:x: viktym<br/>:x: joni-jones<br/>:x: iivashchenko<br/>:x: valdislav<br/>:x: xmav<br/>:x: naydav<br/>:x: olysenko<br/>:x: eug123<br/>:x: heyitsroberthe

cpartica committed:
MAGETWO-62006: Remove @api annotation from new Sales classes - 2.1.3

- removing api annotation

cpartica committed:
MAGETWO-62013: Fix class constructors in a backward compatible way - 2.1.3

- restoring constructors to keep backward incompatibility

Commit:
MAGETWO-62013: Fix class constructors in a backward compatible way - 2.1.3

Commit:
MAGETWO-62006: Remove @api annotation from new Sales classes - 2.1.3

Commit:
MAGETWO-62013: Fix class constructors in a backward compatible way - 2.1.3

Commit:
MAGETWO-62013: Fix class constructors in a backward compatible way - 2.1.3

Commit:
MAGETWO-62013: Fix class constructors in a backward compatible way - 2.1.3

Commit:
Merge remote-tracking branch 'mainline/2.1' into MAGETWO-62013

Commit:
Merge pull request #656 from magento-fearless-kiwis/MAGETWO-62013

[Fiwi] Fix of Backward incompatible changes - 2.1.3

magento-team committed:
MAGETWO-60060: Magento 2.1.3 Publication

 (build 2.1.3.035)

Commit:
Merge pull request #651 from magento-okapis/MAGETWO-58895-compare_products_2.1.x

Fixed issues:
 - MAGETWO-58895: [Backport] Customer is redirected to "Compare Products" Frontend page if tries to remove a Product from comparing - 2.1

Commit:
MAGETWO-62136: Prepare code base for 2.1.4

 Conflicts:
	app/code/Magento/Braintree/view/frontend/web/js/view/payment/method-renderer/paypal.js
	app/code/Magento/Payment/Model/Checks/CanUseForCountry/CountryProvider.php
	app/code/Magento/Payment/Test/Unit/Model/Checks/CanUseForCountry/CountryProviderTest.php
	app/code/Magento/Swatches/view/frontend/web/js/swatch-renderer.js
	composer.json
	composer.lock

Commit:
MAGETWO-62136: Prepare code base for 2.1.4

 - updated composer.lock file

Commit:
MAGETWO-62136: Prepare code base for 2.1.4

 - fixed merge issue

Commit:
MAGETWO-62136: Prepare code base for 2.1.4

 - fixed merge issue

Commit:
Merge pull request #664 from magento-fearless-kiwis/MAGETWO-62136

[Kiwi] Prepare code base for 2.1.4

VenkataUppalapati committed:
MAGETWO-61950: UI Functional Upgrade Test from 2.1.x to 2.x.x. Components Version Support
- add functionality to select minimum compatible version for selected version with sample data
- fix last page bug

VenkataUppalapati committed:
MAGETWO-61950: UI Functional Upgrade Test from 2.1.x to 2.x.x. Components Version Support
- (MAGETWO-59802): Marked Magento\Theme\Model\Config\ValidatorTest testValidateHasRecursiveReference as skipped test due to L2, L4 and Acceptance Tests build failures.

Commit:
Merge pull request #668 from magento-fearless-kiwis/FearlessKiwis-MAGETWO-61950-2.1

Fixed issue:
 - MAGETWO-61950: UI Functional Upgrade Test from 2.1.x to 2.x.x. Components Version Support

Commit:
MAGETWO-60718: [FT] CreateCmsPageEntityTest variations fail on AssertCmsPageForm

Commit:
MAGETWO-60718: [FT] CreateCmsPageEntityTest variations fail on AssertCmsPageForm

Commit:
MAGETWO-62388: Travis Build Fail - 2.1.3

 - fixed integration test isolation for Travis CI

Commit:
MAGETWO-59680: Travis failure: imagettfbbox 2.1.2

 - fixed test failure for Travis CI caused by invalid environment for imagettfbbox

Commit:
Merge pull request #681 from magento-fearless-kiwis/MAGETWO-62388-travis-ci

Fixed issues:
 - MAGETWO-59680: Travis failure: imagettfbbox
 - MAGETWO-62388: Travis Build Fail

RomaKis committed:
MAGETWO-60590:  [FT] CreateCmsBlockEntityTestVariation2 fails to fill form

DianaRusin committed:
Merge branch 'MAGETWO-60718' into 2.1-prs2

ameysar committed:
MAGETWO-62229: There is no Price on product page when product is out of stock

ameysar committed:
MAGETWO-62229: There is no Price on product page when product is out of stock

 - Fix CreateBundleProductEntityTestVariation8

RomaKis committed:
MAGETWO-62428:  After clicking on refund button redirecting on isn't working page

OlgaVasyltsun committed:
Merge remote-tracking branch 'origin/2.1' into 2.1-prs2

OlgaVasyltsun committed:
Merge branch 'MAGETWO-62428' into 2.1-prs2

ameysar committed:
MAGETWO-62229: There is no Price on product page when product is out of stock

 - Unit tests changes

cpartica committed:
MAGETWO-62400: Modules that use repositories in command line tools fail during setup:di:compile 2.1.x

- reverting accessible commands for all modules
- publishing deploy command trough composer

OlgaVasyltsun committed:
Merge remote-tracking branch 'origin/MAGETWO-62229' into 2.1-prs2

cpartica committed:
MAGETWO-62400: Modules that use repositories in command line tools fail during setup:di:compile 2.1.x

- adding comments and setting class names by static accessor

cpartica committed:
MAGETWO-62400: Modules that use repositories in command line tools fail during setup:di:compile 2.1.x

- correct typo for class

cpartica committed:
MAGETWO-62400: Modules that use repositories in command line tools fail during setup:di:compile 2.1.x

- revert usage of class because di compile fails because this is not a class

heyitsroberthe committed:
Merge pull request #695 from magento-fearless-kiwis/MAGETWO-62400-Cli-pre-install

MAGETWO-62400 - Cli pre install

OlgaVasyltsun committed:
MAGETWO-62648: [Github] Parameters from website configuration isn't applied to the corresponding store view #7943

OlgaVasyltsun committed:
MAGETWO-62648: [Github] Parameters from website configuration isn't applied to the corresponding store view #7943

Commit:
MAGETWO-62669: [Backport] [GITHUB] 3rd party payment gateways not visible in adminhtml anymore on 2.1.3 #7891 - 2.1.4

 - refactoring: moved buggy code to separate class

Commit:
MAGETWO-62669: [Backport] [GITHUB] 3rd party payment gateways not visible in adminhtml anymore on 2.1.3 #7891 - 2.1.4

 - covererd with green unit tests

Commit:
MAGETWO-62669: [Backport] [GITHUB] 3rd party payment gateways not visible in adminhtml anymore on 2.1.3 #7891 - 2.1.4

 - added test case for reported bug
 - fixed removal of configuration group if it ordered before special group

Commit:
MAGETWO-62669: [Backport] [GITHUB] 3rd party payment gateways not visible in adminhtml anymore on 2.1.3 #7891 - 2.1.4

 - added test for case when displayIn reference not to special group
 - fixed issue with reference to not special group in displayIn attribute

Commit:
MAGETWO-62669: [Backport] [GITHUB] 3rd party payment gateways not visible in adminhtml anymore on 2.1.3 #7891 - 2.1.4

 - added test for case when several payment methods from one group reference do different special groups
 - fixed implementation issue

Commit:
MAGETWO-62669: [Backport] [GITHUB] 3rd party payment gateways not visible in adminhtml anymore on 2.1.3 #7891 - 2.1.4

 - added docblock

Commit:
MAGETWO-62669: [Backport] [GITHUB] 3rd party payment gateways not visible in adminhtml anymore on 2.1.3 #7891 - 2.1.4

 - fixed failed unit test due to incorrect mock of section modifier

Commit:
MAGETWO-62669: [Backport] [GITHUB] 3rd party payment gateways not visible in adminhtml anymore on 2.1.3 #7891 - 2.1.4

 - fixed unit test namespace

Commit:
Merge pull request #702 from magento-mpi/MAGETWO-62669

[MPI] Fix of payment methods admin panel extensibility

DianaRusin committed:
Merge branch 'MAGETWO-62648' into 2.1-prs2

zakdma committed:
Merge branch '2.1' into 2.1-prs2

ameysar committed:
MAGETWO-62721: "Gift Wrapping for Order Items" setting doesn't work

DianaRusin committed:
Merge branch 'MAGETWO-62721' into 2.1-prs2

DianaRusin committed:
Merge remote-tracking branch 'origin/2.1-prs2' into 2.1-prs2

ameysar committed:
MAGETWO-62428: [Github] Parameters from website configuration isn't applied to the corresponding store view #7943

 - Unit test added

zakdma committed:
Merge branch 'MAGETWO-62428' into 2.1-prs2

ameysar committed:
MAGETWO-62229: There is no Price on product page when product is out of stock

OlgaVasyltsun committed:
MAGETWO-62648: [Github] Parameters from website configuration isn't applied to the corresponding store view

DianaRusin committed:
Merge branch 'MAGETWO-62648' into 2.1-prs2

ameysar committed:
Merge remote-tracking branch 'origin/MAGETWO-62229' into 2.1-prs2

ameysar committed:
MAGETWO-62229: There is no Price on product page when product is out of stock

ameysar committed:
MAGETWO-62428: After clicking on refund button redirecting on isn't working page

ameysar committed:
Merge remote-tracking branch 'origin/MAGETWO-62229' into 2.1-prs2

ameysar committed:
Merge remote-tracking branch 'origin/MAGETWO-62428' into 2.1-prs2

RomaKis committed:
MAGETWO-60590: [FT] CreateCmsBlockEntityTestVariation2 fails to fill form

RomaKis committed:
Merge branch 'MAGETWO-60590' into 2.1-prs2

ameysar committed:
MAGETWO-62229: There is no Price on product page when product is out of stock

RomaKis committed:
Merge remote-tracking branch 'origin/2.1-prs2' into 2.1-prs2

ameysar committed:
Merge remote-tracking branch 'origin/MAGETWO-62229' into 2.1-prs2

ameysar committed:
MAGETWO-60718: [FT] CreateCmsPageEntityTest variations fail on AssertCmsPageForm

ameysar committed:
Merge remote-tracking branch 'origin/MAGETWO-60718' into 2.1-prs2

Commit:
Merge pull request #717 from magento-tsg/2.1-prs2

[TSG] Backporting for 2.1 (prs2)

lanken committed:
MTA-4014: Update extended functional tests suite for Cloud plan

Commit:
Merge pull request #723 from magento-qmt/MTA-4014

[Mavericks] Update extended functional tests suite for Cloud plan

Commit:
MAGETWO-63168: Incorrect configuration tests on Travis CI

Commit:
Merge pull request #749 from magento-east/MAGETWO-63168

MAGETWO-63168: Incorrect configuration tests on Travis CI

Commit:
MAGETWO-63171: Update Magento Zend Framework 1 to latest version

Commit:
Merge remote-tracking branch 'mainline/2.1' into MAGETWO-63171

slavvka committed:
MAGETWO-63329: Travis build is red for 2.1

slavvka committed:
Merge pull request #756 from magento-south/MAGETWO-63171

[SOUTH] MAGETWO-63171: Update Magento Zend Framework 1 to latest version
- MAGETWO-63171 Fix Zend Mail vulnerability - for 2.1.x
- MAGETWO-63329 Travis build is red for 2.1

magento-team committed:
MAGETWO-62135: Magento 2.1.4 Publication

 (build 2.1.4.015)

Commit:
MAGETWO-62137: Prepare code base for 2.1.5-dev

Commit:
MAGETWO-63201: [Backport] - Copyright Year Update 2017 - for 2.1

MomotenkoNatalia committed:
Merge pull request #784 from magento-south/MAGETWO-63201

Fixed issue:
- MAGETWO-63201- Copyright Year Update 2017

Commit:
MAGETWO-64115: Copyright legacy test fails on 2.1

Commit:
MAGETWO-64115: Copyright legacy test fails on 2.1

Commit:
Merge pull request #801 from magento-south/MAGETWO-64115

[SOUTH] MAGETWO-64115: Copyright legacy test fails on 2.1

Commit:
MAGETWO-63201: [Backport] - Copyright Year Update 2017 - for 2.1

Commit:
Merge remote-tracking branch 'mainline/2.1' into MAGETWO-63201

Commit:
MAGETWO-63201: [Backport] - Copyright Year Update 2017 - for 2.1

VladimirZaets committed:
Merge pull request #825 from magento-south/MAGETWO-63201

Bug:
- MAGETWO-63201 [Backport] - Copyright Year Update 2017 - for 2.1

magento-team committed:
MAGETWO-64251: Magento 2.1.5 Publication

 (build 2.1.5.011)

okorshenko commented:
Closing this PR. Incorrect base branch
